### PR TITLE
Docs: updating error codes.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: updating error codes. Closes cksource/ckeditor5-commercial#6869

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 1]
```diff
@@ -1010,13 +1018,25 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 				/**
 				 * Your license does not allow the current distribution channel.
 				 *
+				 * These are the available distribution channels:
+				 * * Self-hosted - the editor is installed via npm or from a ZIP package
+				 * * Cloud - the editor is run from CDN
+				 *
+				 * The licenses available include:
+				 * * GPL license for open-source users
+				 * * Cloud plans (managed via the Customer Portal)
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editor.ts:1027]: Custom plans are also managed by the Customer Portal. Maybe just one group: `Commercial plans (Cloud or sales-assisted)`.

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 2]
```diff
@@ -1010,13 +1018,25 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
...
+				 * * Custom plans (sales-assited)
+				 *
+				 * The relation between distribution channels and licenses works as follows:
 				 * * With a 'GPL' license key, you may use the editor installed via npm or a ZIP package (self-hosted).
 				 * * With the CKEditor Cloud plans, you may use the editor via our CDN.
 				 * * With the CKEditor Custom plans, depending on your plan details, you can use the editor via npm
 				 *   or a ZIP package (self-hosted) or Cloud (CDN)
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editor.ts:1034]:
```suggestion
				 *   or a ZIP package (self-hosted) or Cloud (CDN).
```

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 3]
```diff
@@ -924,9 +924,17 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
...
+				 * Your license key has expired.
+				 *
+				 * If you used the Free Trial, you either need to switch to
+				 * [open-source license](https://ckeditor.com/docs/ckeditor5/latest/getting-started/licensing/license-and-legal.html),
+				 * or acquire one of our [commercial plans](https://ckeditor.com/pricing/) if you wish to keep using the Premium Features.
+				 *
+				 * If you already had one of our Cloud plans, please renew your license in the
 				 * [Customer Portal](https://portal.ckeditor.com).
 				 *
+				 * If you had a Custom plan, please [contact support](https://ckeditor.com/contact/)
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editor.ts:936]:
```suggestion
				 * If you had a Custom plan, please [contact support](https://ckeditor.com/contact/).
```

**Review by Witoso — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 4]
```diff
@@ -924,9 +924,17 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 
 			if ( reason == 'expired' ) {
 				/**
-				 * Your license key has expired. Please renew your license on the
+				 * Your license key has expired.
+				 *
+				 * If you used the Free Trial, you either need to switch to
+				 * [open-source license](https://ckeditor.com/docs/ckeditor5/latest/getting-started/licensing/license-and-legal.html),
+				 * or acquire one of our [commercial plans](https://ckeditor.com/pricing/) if you wish to keep using the Premium Features.
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-core/src/editor/editor.ts:931]: In commit.

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 5]
```diff
@@ -927,14 +927,13 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 				 * Your license key has expired.
 				 *
 				 * If you used the Free Trial, you either need to switch to
-				 * [open-source license](https://ckeditor.com/docs/ckeditor5/latest/getting-started/licensing/license-and-legal.html),
-				 * or acquire one of our [commercial plans](https://ckeditor.com/pricing/) if you wish to keep using the Premium Features.
+				 * [open-source license](https://ckeditor.com/docs/ckeditor5/latest/getting-started/licensing/license-and-legal.html), or
+				 * in case of commercial plan, change the trial key to production or development.
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editor.ts:931]: Just a suggestion 

`trial key to production key or development key.` To make more explicit what we mean.

**Review by Witoso — APPROVED** [review]

File: packages/ckeditor5-core/src/editor/editor.ts [hunk 6]
```diff
@@ -927,14 +927,13 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 				 * Your license key has expired.
 				 *
 				 * If you used the Free Trial, you either need to switch to
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-core/src/editor/editor.ts:929]: The Premium Features Free Trial was always capitalized. Here, well, we may go with "free trial" considering the context.

File: packages/ckeditor5-core/src/editor/editor.ts (hunk 5 above)
Comment (godai78) [reply packages/ckeditor5-core/src/editor/editor.ts]: In commit.

**Review by Witoso — DISMISSED** [review]

**Review by Witoso — APPROVED** [review]

**Pull request merged**